Automattic/studio
# Add Windows Signing

## Proposed Changes

Adds Windows Code Signing (and related tooling to ship dev builds to S3)

Note – this doesn't yet have production build support – that'll be easy to add later, but this is a good place to check in the code and build on further.

## Testing Instructions

1. Note that https://buildkite.com/automattic/studio/builds/318 passes
2. Download the Windows build linked there, validate that it is signed using a Windows computer
